examples/kitchen-sink.yaml

```diff
@@ -11,6 +11,11 @@ file_format: "0.3"
 # Configure if the SDK is disabled or not. This is not required to be provided to ensure the SDK isn't disabled, the default value when this is not provided is for the SDK to be enabled.
 disabled: false
 
+# Configure the log level of the internal logger used by the SDK
+#
+# Environment variable: OTEL_LOG_LEVEL
+log_level: info
```

Comment:
Please add the property description to [type_descriptions.yaml](https://github.com/open-telemetry/opentelemetry-configuration/blob/main/schema/type_descriptions.yaml) to ensure consistency across the examples.

Reply:
Done 👍🏻